neptune/management/internal/api.py

```diff
@@ -13,33 +13,248 @@
+        proxies={}
+    )
+    http_client, _ = create_http_client_with_auth(
+        credentials=credentials,
+        ssl_verify=ssl_verify(),
+        proxies={}
+    )
+    return create_backend_client(
+        client_config=client_config,
+        http_client=http_client
+    )
 
 
-def add_project_member(name, username, role, workspace=None, api_token=None):
-    ...
+@with_api_exceptions_handler
+def get_project_list(api_token: Optional[str] = None) -> List[str]:
+    verify_type('api_token', api_token, (str, type(None)))
 
+    backend_client = _get_backend_client(api_token=api_token)
 
-def get_project_member_list(name, workspace=None, api_token=None) -> typing.Dict[str, str]:
-    ...
+    params = {
+        'userRelation': 'viewerOrHigher',
+        'sortBy': ['lastViewed'],
+        **DEFAULT_REQUEST_KWARGS
+    }
 
+    projects = backend_client.api.listProjects(**params).response().result.entries
 
-def remove_project_member(name, username, workspace=None, api_token=None):
-    ...
+    return list(map(lambda p: f"{p.organizationName}/{p.name}", projects))
 
 
-def get_workspace_member_list(name, api_token=None) -> typing.Dict[str, str]:
-    ...
+@with_api_exceptions_handler
+def create_project(
+        name: str,
+        key: str,
+        workspace: Optional[str] = None,
+        visibility: ProjectVisibility = ProjectVisibility.PRIVATE,
+        description: Optional[str] = None,
+        api_token: Optional[str] = None
+) -> str:
+    verify_type('name', name, str)
+    verify_type('key', key, str)
+    verify_type('workspace', workspace, (str, type(None)))
+    verify_type('visibility', visibility, ProjectVisibility)
+    verify_type('description', description, (str, type(None)))
+    verify_type('api_token', api_token, (str, type(None)))
+
+    backend_client = _get_backend_client(api_token=api_token)
+    project_identifier = normalize_project_name(name=name, workspace=workspace)
+
+    project_spec = re.search(PROJECT_QUALIFIED_NAME_PATTERN, project_identifier)
+    workspace, name = project_spec['workspace'], project_spec['project']
+
+    try:
+        workspaces = backend_client.api.listOrganizations(**DEFAULT_REQUEST_KWARGS).response().result
+        workspace_name_to_id = {f'{f.name}': f.id for f in workspaces}
+    except HTTPNotFound:
+        raise WorkspaceNotFound(workspace=workspace)
+
+    if workspace not in workspace_name_to_id:
+        raise WorkspaceNotFound(workspace=workspace)
+
+    params = {
+        'projectToCreate': {
+            'name': name,
+            'description': description,
+            'projectKey': key,
+            'organizationId': workspace_name_to_id[workspace],
+            'visibility': visibility.value
+        },
+        **DEFAULT_REQUEST_KWARGS
+    }
+
+    try:
+        response = backend_client.api.createProject(**params).response()
+        return f'{response.result.organizationName}/{response.result.name}'
+    except HTTPBadRequest as e:
```

Comment:
I'm pretty sure there are more potential bad request reasons that `ProjectAlreadyExists`... e.g. project key too long?